src/greenpak/i2c.py

```diff
@@ -86,3 +86,42 @@ def read(
             data = self.__i2c.read(byte_count)
         self.__i2c.stop()
         return data if ack else None
+
+
+class GreenPackSMBusAdapter(GreenPakI2cInterface):
+    """An adpater to the Linux 'native' SMBus interface"""
+    import smbus2
+
+    def __init__(self, i2cbusdev="/dev/i2c-0"):
```

Comment:
My thinking is/was, if the value is not provided, assume bus 0 (=device node 0).  A short-hand.
(It isn't really specific, because it is not un-common that if that Linux box/device has an i2c that you can plug your i2c devices easily into, it would / likely be that bus 0 is exposed).

Can be removed, and force user to always pass that device node as you suggest.

Reply:
If you think that this default is common enough than it's good as is.